src/api/LinkingApi.js

```diff
@@ -117,20 +116,20 @@ export function createLinkingEntityType(linkingEntityType :LinkingEntityType) :P
  *   }
  * );
  */
-export function linkEntitySets(linkingEntitySet :LinkingEntitySet) :Promise<> {
+export function linkEntitySets(linkingRequest :LinkingRequest) :Promise<> {
 
   // TODO: everything
 
   let errorMsg = '';
 
-  if (!isValidLinkingEntitySet(linkingEntitySet)) {
-    errorMsg = 'invalid parameter: linkingEntitySet must be a valid LinkingEntitySet';
-    LOG.error(errorMsg, linkingEntitySet);
+  if (!isValidLinkingRequest(linkingRequest)) {
+    errorMsg = 'invalid parameter: linkingRequest must be a valid LinkingRequest';
+    LOG.error(errorMsg, linkingRequest);
     return Promise.reject(errorMsg);
   }
 
   return getApiAxiosInstance(LINKING_API)
-    .post(`/${SET_PATH}`, linkingEntitySet)
+    .post('/', linkingRequest)
```

Comment:
This change breaks the tests. Please fix `testLinkEntitySets()` inside `LinkingApi.test.js`